### PR TITLE
Pil 416 clean

### PR DESCRIPTION
**Pull request by olakunlekuye2**

### CODE REVIEW

File: app/forms/FmContactAddressFormProvider.scala [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+package forms
+
+import forms.mappings.{AddressMappings, Mappings}
+import models.subscription.SubscriptionAddress
+import play.api.data.Form
+import play.api.data.Forms.{mapping, optional}
+
+import javax.inject.Inject
+
+class FmContactAddressFormProvider @Inject() extends Mappings with AddressMappings {
```
Comment (AKKUM) [inline app/forms/FmContactAddressFormProvider.scala:26]: Better to rename SubscriptionAddressFormPorvider and other classes where FmContactAddress name is there.

File: generated-test/controllers/FmContactAddressControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      preAuthenticatedActionBuilders,
+      preDataRetrievalActionImpl,
+      preDataRequiredActionImpl,
+      formProvider,
+      stubMessagesControllerComponents(),
+      viewFmContactAddress
+    )
+
+
+  "FmContactAddress Controller" when {
```
Comment (AKKUM) [inline generated-test/controllers/FmContactAddressControllerSpec.scala:37]: Change name here as well.

File: migrations/FmContactAddress.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+echo "Adding to ViewInstances"
+awk '/trait ViewInstances/ {\
+    print;\
+    print "";\
+    print "   val viewFmContactAddress: FmContactAddressView =";\
+    print "    new FmContactAddressView(pillar2layout, formWithCSRF, govukErrorSummary, govukInput, govukButton)";\
+    next }1' ../test/helpers/ViewInstances.scala > tmp && mv tmp  ../test/helpers/ViewInstances.scala
+
+echo "Migration FmContactAddress completed"
```
Comment (AKKUM) [inline migrations/FmContactAddress.sh:33]: We don't need this file.

Comment (olakunlekuye2) [reply migrations/FmContactAddress.sh:33]: Okay

File: app/forms/FmContactAddressFormProvider.scala (hunk 1 above)
Comment (olakunlekuye2) [reply app/forms/FmContactAddressFormProvider.scala:26]: do you mean refactor anywhere FmContactAddress to  "xyz" i.e. FmContactAddressFormProvider  will become FmxyzFormProvider  . If so, I'm not clear what xyz is.

File: generated-test/controllers/FmContactAddressControllerSpec.scala (hunk 2 above)
Comment (olakunlekuye2) [reply generated-test/controllers/FmContactAddressControllerSpec.scala]: Done

**Pull request closed without merging**